### ISSUE TITLE
Since I updated I can't log in

### ISSUE THREAD

**Issue by litris** [bitwarden/clients#5095]
### Steps To Reproduce

1. Go to xxxx.com
2. Enter email user
3. Click on continue
4. Enter password

### Expected Result

Error:

Ha ocurrido un error.
An unhandled server error has
occurred.

### Actual Result

Error:

Ha ocurrido un error.
An unhandled server error has
occurred.

### Screenshots or Videos

![6404EFDD-88D6-488B-9669-8633C9ECCBE9](https://user-images.githubusercontent.com/2809283/228321340-c4a22f0d-b2f7-4118-b797-5cede08a125e.jpeg)

### Operating System

Linux, iOS

### Operating System Version

Docker synology

### Web Browser

Safari

### Browser Version

iOS 16.2

### Build Version

2023.3.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by litris** [comment]
> You seem to be self-hosting and the error suggests this is a server-side issue. You'll probably want to take this issue over to https://github.com/bitwarden/server or post over at the community forum (https://community.bitwarden.com/) or maybe even contact support.

From there they send me here and you send me there again.

My server is working properly.

I access the home page this happened with the last update.

On the forum I have read more users with the same problem and also after upgrading.

So here I can't ask and they send me to the forum and the forum tells me I can't ask and they send me here.

🤦🏻‍♂️

**Comment by joshtrichards** [comment]
For the record: I don't work for BW. I am just a user that pops in here and contributes from time to time. I imagine an official response will be along at some point. However your issue report doesn't have enough information to reproduce the issue so it's likely you're going to need to talk to support.

Accessing the homepage doesn't mean everything in the backend is working properly. The update may have broken something. I have no idea.

It sounds like you updated the Docker container, correct? That would be the BW server. What BW Docker container do you have deployed? It matters a great deal if you would like help.

Also, just to confirm, you can't log-in via *both* the web client *and* the iOS client?

**Comment by TroyBW** [comment]
Hi there,

I attempted to reproduce your issue and was unable to do so. We use GitHub issues as a place to track bugs and other development related issues. If your issue persists, please contact Bitwarden's support team using our contact form, so we can continue troubleshooting: https://bitwarden.com/contact/ 

You can include a link to this issue in the message content.

The issue here will be closed.

Thanks!

**Comment by litris** [comment]
I had an old version and upgraded to the new version and I can no longer access my server hosted locally on a Synology DSM6.2 Nas in docker.

From the app you can not access and via web if but I put the user and pass and after a while a red message appears.

An error has occurred.
An unhandled server error has
occurred.

I have deleted everything and reinstalled in different ways with the bwdata directory in docker that I made a tutorial many years ago for synology users.

I also deleted it and installed another way from another website and after installing it the message is the same.

I have seen that the error is related to mssql does not start well and when reinstalling I see an error that can not write in the credentials folder or something I do not remember.

I left it for impossible something broke me with this update never activate it only twice since version 1.31.1.

I have a copy of that version but if I put it on it doesn't work.

I do not know what to do I am looking for another system the app works and I have made a copy of the data to move to another system I am a programmer and I see no solution anywhere.

I have changed the permissions of that folder and it gives the same error.

I have already deleted everything.

I do not know if you are aware of this problem think that I never reached the new versions.

**Comment by litris** [comment]
I have reinstalled it from scratch and it does not let me it says that my id and key are not valid.

I put the one I got years ago and it doesn’t work.

I have asked for a new id and key and it doesn’t work either.

I have asked for another one and it doesn’t work either and I get the error “Unable to validate installation id. Problem contacting Bitwarden server.”

image
I have deleted it and what I do is to use the old folder I made as a backup with version 1.32.1.

I have used:

./bitwarden.sh updateself
./bitwarden.sh update

It updated itself and got the usual error when everything started.

But now I was able to take screenshots.

I have left id and key that already had I do not know if the error is for that reason.

From scratch I can not create a new installation because it does not accept any id and key I have created several.

It constantly restarts the container:
bitwarden-mssql

The error it shows is:
/opt/mssql/bin/sqlservr: Error: The secrets directory [/var/opt/mssql/secrets/] could not be created. File: LinuxDirectory.cpp:523 [Status: 0xC0000022 Access Denied errno = 0xD(13) Permission denied].
stderr
10:19:08

I have always worked correctly with version 1.45.2 I should never have upgraded!

It is the same problem that is in this Post happened some time ago in the numbering change is something already known.


 [Update Problem 2022.10.0](https://community.bitwarden.com/t/update-problem-2022-10-0/45665) 
[Password Manager](https://community.bitwarden.com/c/support/pm-ask-the-community/62)
Hallo, when i try to update my self-hosted bitwarden instanz from 2022.9.1 to 2022.10.0, i got the following error: docker: Error response from daemon: Container 6a352c582011060f9e945d59812013babeda78320c1657ca150281e59295ab05 is restarting, wait until the container is running. and docker logs bitwarden-mssql /opt/mssql/bin/sqlservr: Error: The secrets directory [/var/opt/mssql/secrets/] could not be created. File: LinuxDirectory.cpp:523 [Status: 0xC0000022 Access Denied errno = 0xD(13) Per…
Thank you.
